### ISSUE TITLE
Error: Cannot find module

### ISSUE THREAD

**Issue by fabien** [typesense/typesense-js#35]
When requiring the NodeJS module `typesense`, I get this error:

```
Error: Cannot find module '@babel/runtime/helpers/interopRequireDefault'
```

I am now using `dist/typesense` instead.

**Comment by jasonbosco** [comment]
@fabien Thanks for reporting this. This is now fixed in v0.5.1.

**Comment by jasonbosco** [comment]
The root cause of the issue was that @babel/runtime was missing as a dependency in v0.5.0. So v0.5.1 added it as a dependency (instead of a peerDependency). However this bloats the bundle and also causes duplicate inclusion of @babel/runtime across other dependencies that might also need it. 

So with v0.5.2 @babel/runtime is now marked as a peerDependency that you need to install in your app. Running `npm install typesense` should warn you about the peerDepedency.

Installing @babel/runtime in your app will fix the issue you reported above.
